src/Promitor.Core.Scraping/Configuration/Serialization/v1/Model/MetricAggregationV1.cs

```diff
@@ -3,16 +3,16 @@
 
 namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Model
 {
+    /// <summary>
+    /// Contains aggregation settings for a single metric (vs the global aggregation settings).
+    /// </summary>
     public class MetricAggregationV1
     {
-        /// <summary>
-        /// The time period the metric should be aggregated over.
-        /// </summary>
-        public TimeSpan? Interval { get; set; }
+        public AggregationType? Type { get; set; }
```

Comment:
Yeah sorry - I didn't deliberately remove the docs, it's just because of the way that I created duplicate objects and then overwrote the V1 stuff. I'll try to switch it back.

Reply:
No worries, that's what reviews are for!